clients/tv/src/executors/ConsoleExecutor.ts

```diff
@@ -0,0 +1,9 @@
+import CommandEmitter from '../services/CommandEmitter';
+
+function executeFmCommand(command:string, options:Array<string>){
```

Comment:
Szkoda że nie ma naszego ulubionego ESLinta, bo wtedy powiedziałby Ci, że tak powinno to wyglądać:
```typescript
function executeFmCommand(command: string, options: string[]) {
```

Reply:
Nie żeby to był jakiś big deal, ale znasz mnie ;)

Reply:
No i wcięcia są na 4, a nie na 2. ESLint się spruje jak się go podepnie. Trzeba dodać lokalnie .editorconfig, albo zmienić w tym globalnym, że JS i TS jedzie na 2.
Chyba, że chcecie 4 to wtedy nie ma uwagi

Reply:
to prawda, też to zauważyłem, zrobię porządek z linterem w kolejnym kroku

Reply:
W kolejnym PR w sensie?